src/cdk/commands/renderGraphCommandPalette.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+import { PreviewStateMachineCDKWizard } from '../wizards/previewStateMachineCDKWizard'
+import { AslVisualizationCDKManager } from './aslVisualizationCDKManager'
+import { getLogger, Logger } from '../../shared/logger'
+import { renderStateMachineGraphCommand } from './renderStateMachineGraph'
+
+export async function previewCDKStateMachineFromCommandPalette(
```

Comment:
this could live in `/src/cdk/commands/renderStateMachineGraph.ts`. the separation seems arbitrary and that is hinted at by the struggle to find a name for this file (`src/cdk/commands/renderGraphCommandPalette.ts`).